src/core/layout/navbar/responsive-drawer/drawer/drawer-content.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { List } from '@material-ui/core'
+
+import { items } from 'core/layout/navbar/responsive-drawer/drawer/items'
+import { DrawerItem } from 'core/layout/navbar/responsive-drawer/drawer/drawer-item'
+
+export const DrawerContent = () => <List>
+  {
+    items.map((item, index) => <DrawerItem index={index} item={item} key={item} />)
```

Comment:
Let's remove this index, as it is unused.

Reply:
Done